stdlib/test/builtin/test_tuple.mojo

```diff
@@ -0,0 +1,27 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+# RUN: %mojo %s
+
+from testing import assert_true, assert_false
+
+
+def test_tuple_contains():
+    var a = (1, True)
+    assert_true(1 in a)
+    assert_true(a.__contains__(1))
+    assert_true(True in a)
+    assert_false(2 in a)
```

Comment:
Can you please add an `assert_false` case with some different type, e.g. `assert_false("foo" in a)`?